mkosi

```diff
@@ -1329,6 +1391,18 @@ def prepare_tree(args: CommandLineArguments, root: str, do_run_build_script: boo
             cmdline.write(' '.join(args.kernel_command_line))
             cmdline.write("\n")
 
+        dracut_dir = os.path.join(root, "etc/dracut.conf.d")
+        os.mkdir(dracut_dir, 0o755)
+
+        with open(os.path.join(dracut_dir, "mkosi-hostonly.conf"), "w") as f:
+            f.write("hostonly=no\n")
+
+        with open(os.path.join(dracut_dir, "mkosi-systemd-extras.conf"), "w") as f:
+            f.write(f"install_optional_items+={' '.join(DRACUT_SYSTEMD_EXTRAS)}\n")
```

Comment:
```suggestion
            f.write(f'install_optional_items+=" {" ".join(DRACUT_SYSTEMD_EXTRAS)} "\n')
```
dracut wants spaces at the start and end of these strings, since they use them as arrays and I guess they want to be save about splitting elements. 

Alternatively this could be written as
```python
for arg in DRACUT_SYSTEMD_EXTRAS:
    f.write(f'install_optional_items+=" {arg} "\n')
```
To make it easier to read, because the line is over 270 characters long.